packages/theme/bin/generate-primitive-tokens/index.ts

```diff
@@ -30,28 +29,24 @@ const __dirname = path.dirname( __filename );
 const colorJsonPath = path.join( __dirname, '../../tokens/color.json' );
 
 const transformColorStringToDTCGValue = ( color: string ) => {
-	if ( /oklch|p3/.test( color ) ) {
-		let parsed: PlainColorObject;
-		try {
-			parsed = to( parse( color ), OKLCH );
-		} catch {
-			return color;
-		}
-
-		const coords = parsed.coords;
-		return {
-			colorSpace: 'oklch',
-			components: [
-				Math.floor( 10000 * coords[ 0 ] ) / 10000, // l
-				coords[ 1 ], // c
-				isNaN( coords[ 2 ] ) ? 0 : coords[ 2 ], // h
-			],
-			...( parsed.alpha < 1 ? { alpha: parsed.alpha } : undefined ),
-			hex: serialize( to( parsed, sRGB ), { format: 'hex' } ),
-		};
+	let parsed: PlainColorObject;
+	try {
+		parsed = to( parse( color ), sRGB );
+	} catch {
+		return color;
 	}
```

Comment:
Not introduced in this PR, but it's not obvious to me what kinds of cases would error on the `parse` but is still safe to silently return with the original value.